src/ui/MLFacet/MLFacet.ts

```diff
@@ -39,7 +39,18 @@ export interface IMLFacetOptions extends IResponsiveComponentOptions {
 }
 
 /**
- * Renders a facet in the search interface.
+ * The `MLFacet` component displays a *facet* of the results for the current query. A facet is a list of values for a
+ * certain field occurring in the results, ordered using a configurable criteria (e.g., number of occurrences).
+ *
+ * The list of values is obtained using an array of [`FacetRequest`]{@link IFacetRequest} operation performed at the same time
+ * as the main query.
+ *
+ * The `MLFacet` component allows the end user to drill down inside a result set by restricting the result to certain
```

Comment:
end user -> end-user

I'd remove the second sentence (the one about value filtering and facet search). These features are not implemented yet for MLFacet, right?